src/nrc/api/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+from django.utils.html import format_html
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.models import JWTSecret
+from zds_client import ClientAuth
+
+admin.site.unregister(JWTSecret)
+
+
+@admin.register(JWTSecret)
+class JWTSecretAdmin(admin.ModelAdmin):
+
+    list_display = ("identifier",)
+
+    readonly_fields = ("get_jwt",)
+
+    @admin.display(description="jwt")
+    def get_jwt(self, obj):
+        if obj.identifier and obj.secret:
+            auth = ClientAuth(obj.identifier, obj.secret)
+            jwt = auth.credentials()["Authorization"]
+            return format_html(
+                '<code class="copy-action jwt" data-copy-value="{val}">{val}</code><p>{hint}</p>',
```

Comment:
There are some references to a copy action here in the class and the data attribute, but those aren't used, so let's either implement the copy functionality or remove these references

Reply:
I don't think this functionality is even in OpenZaak, so it probably makes sense to remove it